_topic_map.yml

```diff
@@ -119,6 +119,23 @@ Topics:
   - Name: Autoscaling worker nodes
     File: rosa-autoscale-nodes
 ---
+Name: Monitoring user workloads
+Dir: monitoring
```

Comment:
Name should be Monitoring
since the section contains cluster monitoring and user workloads monitoring